packages/cli/lib/services/verification.service.ts

```diff
@@ -93,6 +94,32 @@ class VerificationService {
         }
     }
 
+    public async runMigration(loadLocation: string): Promise<void> {
+        if (process.env['NANGO_CLI_UPGRADE_MODE'] === 'ignore') {
+            return;
+        }
+        const localConfig = await loadLocalNangoConfig(loadLocation);
+
+        if (!localConfig) {
+            return;
+        }
+
+        const version = determineVersion(localConfig);
+        if (version === 'v2') {
+            console.log(chalk.blue(`nango.yaml is already at v2.`));
+        }
+        if (version === 'v1' && localConfig.integrations) {
+            exec(`node ${getNangoRootPath()}/scripts/v1-v2.js ./${nangoConfigFile}`, (error) => {
```

Comment:
would it make sent to have the migration logic part of `cli/lib` so it can be called here without having to run node within node. The script can also import the logic